pkgs/development/compilers/idris2/default.nix

```diff
@@ -0,0 +1,54 @@
+# racket is busted: https://github.com/NixOS/nixpkgs/issues/11698
+
+{ stdenv, fetchFromGitHub, makeWrapper, runCommand
+, idris, chez, racket, chickenPkgs
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "idris2";
+
+  src = fetchFromGitHub {
+    owner = "edwinb";
+    repo = "Idris2";
+    rev = "bf5b2298e068ebeaddd70b695426f52ebe67c1ab";
+    sha256 = "1ixmdy4zbllmpwmvmmqv1rj9a2y243pggf42pdqkvrkjw50zqlyj";
+  };
+
+  version = "0.0.0";
```

Comment:
Nixpkgs typically uses the YYYY-MM-DD schema for untagged sources.

Reply:
Ok, updated to date-based version format.